setup.cfg

```diff
@@ -15,3 +17,8 @@ test=pytest
 [isort]
 forced_separate=evalml
 multi_line_output=3
+include_trailing_comma = True
```

Comment:
Just wondering, did you also try this? https://pycqa.github.io/isort/docs/configuration/black_compatibility/

Looks like we can set profile=black?

Reply:
Ah, looks like we need isort >=5. Wonder if it's worth upgrading to not have to worry about the config?

Reply:
Oooo, I didn't know about this--this is really cool!! Looks like isort 5.0.0 was a big change (https://github.com/PyCQA/isort/blob/main/CHANGELOG.md) but I'd be down to try and see what changes it might cause / whether or not we like them :)

Reply:
Opened https://github.com/alteryx/evalml/pull/2334 to see the changes, looks like the main change is that submodules are sorted alphabetically, which is quite nice. I'd say it's a good idea to upgrade!

Reply:
Thanks @angela97lin ! I agree!!

Reply:
Ugh, it looks like we have to ignore the __init__.py files though, since otherwise we run into import errors (circular dependencies and all that good stuff). Makes me wonder if we have the proper structure for all the __init__ files but adding `skip=__init__.py` for the isort config for now.